mreg/api/permissions.py

```diff
@@ -364,9 +364,8 @@ def has_ipaddress_permission(self, request: Request, view: ListCreateAPIView, va
         try:
             ipaddr = ipaddress.ip_address(ip)
         except ValueError:
-            raise exceptions.ValidationError(
-                {"ipaddress": "Invalid IP address."},
-            )
+            # invalid IP, let serializer handle it
+            return True 
```

Comment:
This might be a bit contentious...?

I think it makes more sense to pass the request on to other permission classes, and eventually the serializer, as opposed to throwing a 403 here.